core/fields/Field_Rich.php

```diff
@@ -253,11 +307,10 @@ public function display() {
 									});
 									helptext += "</ul>";
 								}
-								let hL = [helptext, ""];
-
+								
 								function onCreate() {
 									// insert link html at anchor location
-									let link_html = "<a id='newly_created_link_for_<?php echo $this->name;?>'>" + selection + "</a>";
+									let link_html = "<a data-foo='bar' target='"+target+"' class='"+classes+"' id='newly_created_link_for_<?php echo $this->name;?>'>" + selection + "</a>";
```

Comment:
any reason not to use `${varaible}text` sysntax here? cleaner

Reply:
previous code uses concat so continued the party. will change.